src/tests/ftest/recovery/start_subcommand.py

```diff
@@ -0,0 +1,164 @@
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=recovery,cat_recov,start_subcommand
+        :avocado: tags=StartSubcommandTest,test_reset
+        """
+        # 1. Create a pool.
+        self.log_step("Create a pool")
+        pool = self.get_pool(connect=False)
+
+        # 2. Inject orphan pool fault.
+        self.log_step("Inject orphan pool fault")
+        dmg_command = self.get_dmg_command()
+        dmg_command.faults_mgmt_svc_pool(
+            pool=pool.identifier, checker_report_class="CIC_POOL_NONEXIST_ON_MS")
+
+        # 3. Start the checker with interactive mode.
+        self.log_step("Start the checker with interactive mode.")
+        dmg_command.check_enable()
+        dmg_command.check_set_policy(all_interactive=True)
+        dmg_command.check_start()
+
+        # 4. Verify that the orphan pool is detected.
+        self.log_step("Verify that the orphan pool is detected.")
+        repair_reports = None
+        for _ in range(8):
+            check_query_out = self.get_dmg_command().check_query()
+            if check_query_out["response"]["status"] == "RUNNING":
+                repair_reports = check_query_out["response"]["reports"]
+                break
+            time.sleep(5)
+        if not repair_reports:
+            self.fail("Checker didn't detect any inconsistency!")
+        fault_msg = repair_reports[0]["msg"]
+        orphan_pool = "orphan pool"
+        if orphan_pool not in fault_msg:
+            msg = (f"Checker didn't detect the {orphan_pool} (1)! Fault msg = "
+                   f"{fault_msg}")
+            self.fail(msg)
+
+        # 5. Stop the checker.
+        self.log_step("Stop the checker.")
+        dmg_command.check_stop()
+
+        # 6. Remove the pool directory from the mount point.
+        self.log_step("Remove the pool directory from the mount point.")
+        scm_mount = self.server_managers[0].get_config_value("scm_mount")
+        pool_path = f"{scm_mount}/{pool.uuid.lower()}"
+        pool_out = check_file_exists(
+            hosts=self.hostlist_servers, filename=pool_path, sudo=True)
+        if not pool_out[0]:
+            msg = ("MD-on-SSD cluster. Contents under mount point are removed by control "
+                   "plane after system stop.")
+            self.log.info(msg)
+            dmg_command.system_start()
+            # return results in PASS.
+            return
+        command = f"sudo rm -rf {pool_path}"
+        remove_result = run_remote(
+            log=self.log, hosts=self.hostlist_servers, command=command)
+        if not remove_result.passed:
+            self.fail(f"Failed to remove {pool_path} from {self.hostlist_servers}")
+        success_nodes = remove_result.passed_hosts
+        if self.hostlist_servers != success_nodes:
+            msg = (f"Failed to remove pool directory! All = {self.hostlist_servers}, "
+                   f"Success = {success_nodes}")
+            self.fail(msg)
```

Comment:
If we get here, that means `remove_result.passed == True`, which also means `remove_result.passed_hosts == self.hostlist_servers`. So we don't need this check at all.
```suggestion
```

Reply:
Done